Makefile

```diff
@@ -143,7 +143,12 @@ endif
 install: build.common
 	@$(MAKE) go.install
 
-check test: ## Runs unit tests.
+
```

Comment:
```suggestion
```
Apart from this, everything looks good to me, and one small thing, I don't know why that `checkmake` sounds `checkmate` to me in the back of my mind  😄